tests/functional/test_no_color.py

```diff
@@ -0,0 +1,61 @@
+"""
+Test specific for the --no-color option
+"""
+import os
+import platform
+import subprocess as sp
+import sys
+
+import pytest
+
+
+@pytest.mark.skipif(sys.platform == 'win32',
```

Comment:
A cross-platform test would be nice to have.

Reply:
@pradyunsg , I would love that too, but it is unfair to ask me to implement this. I have no access to
a windows machine.

Reply:
No issues then. :)